src/inet/tests/TestInetLayerDNS.cpp

```diff
@@ -339,6 +340,8 @@ static void TestDNSResolution_NoHostRecord(nlTestSuite * testSuite, void * testC
             false
         }
     );
+    #endif
```

Comment:
nit: align #endif with above? 

Not sure if this would be fighting the autoformatter :( looks odd though.